### DISCUSSION TITLE
Is DRC being removed?

### DISCUSSION THREAD

**Discussion by SyCoREAPER** [deniscerri/ytdlnis#604] [Q&A] [answered]
It's come to my attention that "other apps" cannot see/play non-DRC (volume leveling) tracks, it can only be disabled with the official YT app. Is this removing the normalization effect that is now/currently forced by default?

**Comment by error-reporting** [comment, marked as answer]
Standard non drc audio track are not removed
separate non drc and drc track still exists

**Reply by SyCoREAPER** [reply]
Is there a way to force non-DRC tracks to appear? They don't always do so on here despite other apps (I'm trying not to name drop or go OT) showing there being DRC and non-DRC tracks.

**Reply by error-reporting** [reply]
give me the URL

**Reply by SyCoREAPER** [reply]
I'll have to find one again. It was a while ago when I came across this for the first time. I'm solely going off of memory at the moment.

**Reply by error-reporting** [reply]
is you able to find one

**Reply by error-reporting** [reply]
hey @SyCoREAPER
